src/project.cc

```diff
@@ -429,6 +433,27 @@ std::vector<Project::Entry> LoadCompilationEntriesFromDirectory(
   unsigned int num_commands = clang_CompileCommands_getSize(cx_commands);
   clang_time.Pause();
 
+  // Passing -working-directory to libclang does not have any effect on Windows. 
+  // As a result, libclang is unable to correctly resolve include directories 
+  // that are defined with a relative path in compile_commands.json. To work
+  // around this, we change the working directory to the directory we pass to
+  // to -working-directory (without -working-directory libclang defaults to the
+  // working directory of the process).
+#ifdef _WIN32
+  if (num_commands > 0) {
```

Comment:
We should validate all comments have the same working directory. If they don't we should emit an error using LSP api so the user sees it.

Reply:
you mean commands?

Reply:
Do I change the working directory in this case?

Reply:
Like this: https://github.com/cquery-project/cquery/blob/master/src/match.cc#L35

Reply:
When there are multiple directories do we want to change the working directory regardless (for example to the first directory) and continue or do we want to stop the server? (or not change directory and continue).

Reply:
I think we'd have to just pick one and keep going.